help/md/dnCDBDP.md

```diff
@@ -1,9 +1,73 @@
 ## name
 dnCDBDP
 ## title
+Character Dependent Birth Death Process
 ## description
+This function simulates a tree under a character-dependent birth-death process.
 ## details
+This function is a flexible simulator that can be used for several phylogenetic models.
```

Comment:
Again, listing the argument definitions doesn't count as documentation, since we already have that information.